brainframe_qt/ui/brainframe_app.py

```diff
@@ -194,6 +198,10 @@ def _start_ui(self):
             if server_version != brainframe_qt.__version__:
                 self._handle_version_mismatch(server_version)
 
+            # Initialize the StreamManager
+            # TODO: Find a better way of doing this
```

Comment:
Ima keep it real with u chief!! Considering the circumstances, I can't think of a better way to do this.